plugins/woocommerce-blocks/tests/e2e/tests/related-products/related-products.block_theme.spec.ts

```diff
@@ -63,4 +63,54 @@ test.describe( `${ blockData.name } Block`, () => {
 			await editor.getBlockByName( blockData.slug )
 		).toBeVisible();
 	} );
+
+	test( "doesn't display additional Products (Beta) options", async ( {
+		page,
+		admin,
+		editor,
+		editorUtils,
+	} ) => {
+		await admin.visitSiteEditor( {
+			postId: `woocommerce/woocommerce//single-product`,
+			postType: 'wp_template',
+		} );
+		await editorUtils.enterEditMode();
+		const relatedProducts = await editorUtils.getBlockByName(
+			blockData.slug
+		);
```

Comment:
You can use `editor` here:

```suggestion
		const relatedProducts = await editor.getBlockByName(
			blockData.slug
		);
```